### PR TITLE
libxcrypt transition - CI test, DO NOT MERGE

### PR DESCRIPTION
**Pull request by akhuettel**

### CODE REVIEW

File: profiles/base/package.use.force [hunk 1]
```diff
@@ -91,12 +91,13 @@ sys-libs/libcxx libcxxabi
 # the flag.
 sys-kernel/vanilla-kernel initramfs
 
+# Andreas K. Hüttel <dilfridge@gentoo.org> (2021-07-14)
 # Sergei Trofimovich <slyfox@gentoo.org> (2019-11-06)
```
Comment (thesamesam) [inline profiles/base/package.use.force:95]: Drop other authors -- pkgdev and other parsers can't handle it, and we have git history anyway. Or just put toolchain@

File: profiles/base/package.use.mask [hunk 2]
```diff
@@ -286,10 +286,14 @@ dev-python/statsmodels doc
 # broken static-linking in glibc and maybe others
 >app-shells/mksh-57-r1 static
 
+# Andreas K. Hüttel <dilfridge@gentoo.org> (2021-07-14)
```
Comment (thesamesam) [inline profiles/base/package.use.mask:289]: See previous authors commment

File: profiles/base/package.use.mask [hunk 3]
```diff
@@ -286,10 +286,14 @@ dev-python/statsmodels doc
...
 # Patrick McLean <chutzpah@gentoo.org> (2019-11-07)
-# Collides with sys-libc/glibc[crypt]
-# (NOTE: Likely needs to be moved to musl + uclibc profiles, not dropped entirely)
```
Comment (thesamesam) [inline profiles/base/package.use.mask:291]: Let's do that

File: profiles/package.mask [hunk 4]
```diff
@@ -651,11 +651,6 @@ kde-apps/kdebase-meta:5
 # Mask for testing revdeps.
 >=dev-games/mygui-3.4.0
 
-# Patrick McLean <chutzpah@gentoo.org> (2020-02-07)
-# Mask until sys-libs/libxcrypt[system] is unmasked
-# (NOTE: Likely needs to be moved to musl + uclibc profiles, not dropped entirely)
```
Comment (thesamesam) [inline profiles/package.mask:656]: Do this

**Review by thesamesam — CHANGES_REQUESTED** [review]
Not yet tested (I want to do an upgrade from a stage3 first with this applied to the local tree), just making comments on mask changes first

**Comment by akhuettel** [comment]
All in the tree.

**Pull request closed without merging**